jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -364,7 +380,9 @@ ecma_utf8_string_to_number_by_radix (const lit_utf8_byte_t *str_p, /**< utf-8 st
  */
 ecma_number_t
 ecma_utf8_string_to_number (const lit_utf8_byte_t *str_p, /**< utf-8 string */
-                            lit_utf8_size_t str_size) /**< string size */
+                            lit_utf8_size_t str_size, /**< string size */
+                            uint32_t allow_underscore_option) /**< true - if '_' characters are allowed in the string
```

Comment:
The comment is invalid. Also the name should simply be `options` since we may add more options in the future.